src/System/i18n/locales/en-US/translation.json

```diff
@@ -1,24 +1,30 @@
 {
-  "navbar":{
-    "show":"Show",
-    "shows":"Shows",
-    "buy":"Buy",
-    "artists":"Artists",
-    "artworks":"Artworks",
-    "galleries":"Galleries",
-    "auctions":"Auctions",
-    "fairs":"Fairs",
-    "museums":"Museums",
-    "viewingRooms":"Viewing Rooms",
-    "nfts":"NFTs",
-    "signup":"Sign Up",
-    "login":"Log In",
-    "sell":"Sell",
-    "priceDatabase":"Price Database",
-    "editorial":"Editorial",
-    "downloadApp":"Download Apps",
+  "specialists": {
```

Comment:
pls alphabetize all keys before this all gets out of hand 🙏 

Also, this makes me wonder if we can use something other than `.json` here so that we can add comments; it'd be good to be able to add something like:

```
// /meet-the-specialists
"specialists: { ...
```

so people know where the copy ends up living. But that's for another day